pdfslick/pdfslick
# pdfjs-dist upgrade and highlight annotation support added

* Upgrade pdfjs-dist to latest
* Create css script for combining pdfjs-dist/web/pdf_viewer.css with custom css.  Also inline svgs in final output.
* Remove image copies since they get inlined in final pdf_viewer.css.
* Ensure build works cross platform by using copyfiles.
* Add HighlightMenu to toolbar in solidweb.
* Add highlightDefaultColor to state and updateHighlightDefaultColor method to PdfSlick.
* Add overflow auto to fix overflow issues.

I have tested basic PDF functionality in solid, react, and svelte.  There may be some things that are broken that I am not aware of and more thorough testing may be necessary.
